ackermann_steering_controller/CMakeLists.txt

```diff
@@ -49,13 +49,23 @@ install(FILES ackermann_steering_controller_plugins.xml
 
 if (CATKIN_ENABLE_TESTING)
   find_package(catkin REQUIRED COMPONENTS
+    controller_interface
```

Comment:
only compile-time dependencies should be listed here

Reply:
Tried to fix at https://github.com/ros-controls/ros_controllers/pull/380/commits/e745c484517543b00b25111f5aa9dfe749c236ed looking at test codes inside